tests/Tests.hx

```diff
@@ -23,6 +23,21 @@ using StringTools;
 using buddy.Should;
 using dataclass.Converter;
 
+class CompileTests
+{
+	macro static public function missing_id_in_RequireId_compiled() {
+		try {
+			haxe.macro.Context.typeExpr(macro new RequireId( {} ));
```

Comment:
I tested this works by adding `{id: 1}` and it returns true.  But I see now that this might not be obvious because it could fail to compile for other reasons